modules/Emails/EmailUI.php

```diff
@@ -473,6 +473,7 @@ public function populateComposeViewFields($bean = null, $emailField = 'email1',
                     foreach($addresses as $address) {
                         if($address['email_address'] == $myBean->{$emailField} && (int)$address['opt_out']) {
                             $optOut = true;
+                            $emailLink .= '<span class="email-line-through">';
```

Comment:
where is the closing tag. or is it a fix for a missing opening tag somewhere?

Reply:
The email1 field does not have a closing tag. It is added later in  a few places eg customCode in listviewdefs . However, we do not have a means to communicate that we need a closing span tag ... So I have refactored the code to support having the closing tag by placing the responsibility of generating the links to EmailUI::populateComposeViewFields() .